samples/azure-quantum/entanglement-swap/ES-quantinuum-qiskit.ipynb

```diff
@@ -0,0 +1,930 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job id ea87415f-256d-11ed-92be-dc215ca5fadb\n"
+     ]
+    }
+   ],
+   "source": [
+    "validator_backend = provider.get_backend(\"quantinuum.hqs-lt-s2-apival\")\n",
+    "qc_t = transpile(qc, validator_backend)\n",
+    "job = validator_backend.run(qc_t, job_name=\"Validate Entanglement Swapping\")\n",
+    "job_id = job.id()\n",
+    "print(\"Job id\", job_id)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job Status: job has successfully run\n"
+     ]
+    }
+   ],
+   "source": [
+    "job_monitor(job)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "API Validation succeeded!\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = job.result()\n",
+    "if not result.success:\n",
+    "    raise Exception(\"API validation Failed\")\n",
+    "print(\"API validation succeeded!\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now with the confidence that our job will work we can submit it to the simulator.  The Quantinuum emulator provided by Azure Quantum differs from the simulator running locally, in that it accurately models noise and behavior of the Quantinuum quantum computer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sim_backend = provider.get_backend(\"quantinuum.hqs-lt-s2-sim\")\n",
+    "qc_t = transpile(qc, validator_backend, job_name=\"Simulate Entanglement Swapping\")\n",
+    "job = sim_backend.run(qc_t)\n",
+    "job_id = job.id()\n",
+    "print(\"Job id\", job_id)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_monitor(job)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = job.result()\n",
+    "print(result)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "counts = {format(n, \"03b\"): 0 for n in range(8)}\n",
+    "counts.update(result.get_counts(qc_t))\n",
+    "print(counts)\n",
+    "plot_histogram(counts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that while quite good, we do get some noise from running our code on the emulator that accurately mimics a quantum device. With larger circuits such as the ones below this problem will only get worse.\n",
```

Comment:
The first sentence here is a bit awkward? I may also suggest dropping or moderating the second sentence a bit to sound a bit less negative.